src/app/pages/SearchResultsPage/hooks.ts

```diff
@@ -89,32 +59,17 @@ export function useTransactionsConditionally(
 export function useRuntimeAccountConditionally(
   address: string | undefined,
 ): ConditionalResults<RuntimeAccount> {
-  const queries = [
-    useGetRuntimeAccountsAddress(Network.mainnet, Runtime.emerald, address!, {
-      query: {
-        enabled:
-          !!address && RouteUtils.getEnabledLayersForNetwork(Network.mainnet).includes(Runtime.emerald),
-      },
-    }),
-    useGetRuntimeAccountsAddress(Network.mainnet, Runtime.sapphire, address!, {
-      query: {
-        enabled:
-          !!address && RouteUtils.getEnabledLayersForNetwork(Network.mainnet).includes(Runtime.sapphire),
-      },
-    }),
-    useGetRuntimeAccountsAddress(Network.testnet, Runtime.emerald, address!, {
-      query: {
-        enabled:
-          !!address && RouteUtils.getEnabledLayersForNetwork(Network.testnet).includes(Runtime.emerald),
-      },
-    }),
-    useGetRuntimeAccountsAddress(Network.testnet, Runtime.sapphire, address!, {
-      query: {
-        enabled:
-          !!address && RouteUtils.getEnabledLayersForNetwork(Network.testnet).includes(Runtime.sapphire),
-      },
-    }),
-  ]
+  const queries = RouteUtils.getEnabledScopes()
+    .filter(scope => scope.layer !== Layer.consensus)
+    .map(scope =>
+      // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
`// eslint-disable-next-line react-hooks/rules-of-hooks -- copy an explanation`

Reply:
OK